docs/DeveloperGuide.adoc

```diff
@@ -729,27 +743,125 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |teacher |add new timeslot in timetable |I can plan my teaching schedule
+
+|`* * *` |teacher |add new location for a module in timetable |I can plan my teaching schedule
```

Comment:
How is adding a new location help in planning the teaching schedule? Maybe you meant venue?

Reply:
thanks, changed accordingly